desktop/libs/notebook/src/notebook/api.py

```diff
@@ -405,7 +405,7 @@ def cancel_statement(request):
   response = {'status': -1}
 
   notebook = json.loads(request.POST.get('notebook', '{}'))
-  snippet = None
+  snippet = json.loads(request.POST.get('snippet', '{}'))
```

Comment:
is `None` not possible instead of `'{}'` as a default? I'm worried there might be unwanted side effects in the _get_snippet code.

Reply:
@Harshg999 and I have discussed this thing and looking good to us. @JohanAhlen